### ISSUE TITLE
Unable to grant permission to intercept requests

### ISSUE THREAD

**Issue by designosis** [firephp/firephp-for-firefox-devtools#73]
I'm using the "FirePHP (Official)" plugin (v1.5.4) for Chrome, downloaded from the Chrome web store.

I load any page (e.g. firephp.org), click the firebug extension, and check the "give permissions box".

![Screen Shot 2022-03-16 at 3 38 47 PM](https://user-images.githubusercontent.com/146623/158695801-95085770-7f82-42d2-8b49-e5313afc9d32.png)

At this point, after clicking away from that checkbox dialog, clicking back in shows the box unchecked. No matter what I do, there's no persistence on that checkbox, it always returns to unchecked status, and F12 shows ...

![Screen Shot 2022-03-16 at 3 39 06 PM](https://user-images.githubusercontent.com/146623/158696019-bb66204e-2ebb-48a6-940c-c4f1b747f789.png)

I've disable every other extension to make sure it's not a conflict, and this fixes nothing. 
What am I doing wrong?

**Comment by stSaHiB** [comment]
Hi, welcome to the best debugging tool I know (and we tried a lot, still we are using 95% firePHP and 5% xdebug for difficult problems)

I got it installed on FireFox (of course!), Chromium, Chrome (latest: 99.0.4844.74) and Edge. its working everywhere.

When you check that grant permission box, there should be a popup in Chrome asking for additional rights. Do you get that message and do you grant the rights?
![grafik](https://user-images.githubusercontent.com/761264/158712856-769a603a-5911-43d9-9638-3a0956c8afd6.png)
If you dont grant em, it wont work...

**Comment by designosis** [comment]
Thanks for the response!

I get no such pop-up after clicking the checkbox, but thanks to your input now I know why.

On OSX, I have multiple Chrome windows, each with many open tabs. Depending on the project I'm working on (and in order to keep from going crazy), I use only ONE of those windows at a time, and HIDE the others (switching between them as I change projects).

What's happening is this: when I click the FirePHP extension icon and click the checkbox, the notification box (from your image above) appears in one of my HIDDEN windows! I had to open them all one-by-one until I saw the notification you described.

I have no idea why Chrome would open a notification dialog box in a hidden window, rather than in the one I'm currently on ... but there it is.

Thanks :)
